### PR TITLE
Search

### PR DESCRIPTION
**Pull request by nikolaswise**

### CODE REVIEW

File: lib/usage/index.js [hunk 1]
```diff
@@ -21,50 +21,53 @@ export default function (constructor = {}) {
...
+      // })
     },
-
     billing: function (options = {}) {
       return this.request(`/subscription`, options, false, 'https://billing.arcgis.com/sms/rest')
     },
-
-    flattenData: function (response) {
-      var _usage = this
+    interperetData: function (response) {
```
Comment (kneemer) [inline lib/usage/index.js]: interpretData ?

Comment (nikolaswise) [inline lib/usage/index.js]: Usage API returns things like `count: 8000' and `stype: 'nacfroute'` and other such garbage. This method will turn that into english at some point. Super stubby right now.

Comment (kneemer) [inline lib/usage/index.js]: I just meant the spelling :)

Comment (nikolaswise) [inline lib/usage/index.js]: lol

**Pull request merged**